packages/wrangler/src/index.tsx

```diff
@@ -86,13 +90,16 @@ async function readConfig(configPath?: string): Promise<Config> {
     "durable_objects",
     "experimental_services",
   ];
-  Object.keys(config.env || {}).forEach((env) => {
+  Object.entries(environments).forEach(([envName, envValue]) => {
+    if (envValue === undefined) {
+      return;
```

Comment:
same here, I think we don't want to return early

Reply:
Similar to https://github.com/cloudflare/wrangler2/pull/233#discussion_r783082960 - the previous code would crash in this case.

Reply:
should it be an error in both cases then?
`throw new Error('Missing ${something} in environments)`